.aws/src/iam.ts

```diff
@@ -57,7 +164,15 @@ export class AgentIamPolicies extends Construct {
         'ecs:RegisterTaskDefinition',
         'ecs:ListTaskDefinitions',
         'ecs:DescribeTaskDefinition',
-        'ecs:DeregisterTaskDefinition'
+        'ecs:DeregisterTaskDefinition',
+        'ec2:DescribeVpcs',
+        'ec2:CreateNetworkInterface',
+        'ec2:DescribeNetworkInterfaces',
+        'ec2:DeleteNetworkInterface',
+        'ec2:AssignPrivateIpAddresses',
+        'ec2:UnassignPrivateIpAddresses',
+        'ec2:DescribeSubnets',
+        'ecs:DescribeTasks'
```

Comment:
agent need ec2 permissions to submit an ECS Fargate task that uses Vpc networking